ChameleonMiniGUI/FrmMain.cs

```diff
@@ -2141,12 +2152,12 @@ internal void DownloadAndSaveDump(string filename)
 
             // Also check if the tag is UL to save the counters too
             var configStr = SendCommand($"CONFIG{_cmdExtension}?").ToString();
-            if (!string.IsNullOrWhiteSpace(configStr) && (configStr.Contains("ULTRALIGHT")))
+            if (!string.IsNullOrWhiteSpace(configStr) && (configStr.Contains("ULTRALIGHT_EV1")))

```

Comment:
Will this not break backward compability?

Reply:
It is used to download from chameleon some extra pages (52 bytes) with extra data. Simple ultralights has no extra data, no need to download more bytes